src/main/java/com/mathworks/ci/helper/MatlabBuild.java

```diff
@@ -57,11 +57,14 @@ default File getTempWorkingDirectory() {
 
     default String getPlatformSpecificRunner(File tempDirectory) throws IOException {
         if (SystemUtils.IS_OS_WINDOWS) {
```

Comment:
One option if we changed how `copyFileInWorkspace` worked (added benefit of less returns)
```
        if (SystemUtils.IS_OS_WINDOWS) {
            sourceFile = "win64\\run-matlab-command.exe";
            targetFile = tempDirectory + "\\run-matlab-command.exe";
        } else if (SystemUtils.IS_OS_MAC) {
            sourceFile = "maci64/run-matlab-command";
            targetFile = tempDirectory + "/run-matlab-command";
        } else {
            sourceFile = "glnxa64/run-matlab-command";
            targetFile = tempDirectory + "/run-matlab-command";
        }
        copyFileInWorkspace(sourceFile, targetFile);
        return targetFile
```